vue/src/components/loki/LogLabelValuesCont.vue

```diff
@@ -5,7 +5,7 @@
       v-for="(item, idx) in labels"
       :key="idx"
       v-model="item.selected"
-      :attr-key="item.name"
+      :attr-key="item.value"
```

Comment:
Is this correct? `attr-key` meant to be `label.name` - perhaps rename `attr-key` to `label-name` or `label-value` for clarity?